README.md

```diff
@@ -118,6 +121,24 @@ Gates
 * `time_range` &mdash; enable a feature for the duration of a time range
 * `percentage_of_time` &mdash; enable a feature for a given percentage of checks
 * `rails_env` &mdash; enable a feature for a specified Rails environment
+* `group` &mdash; enable a feature for a specified group of users (needs additional configuration)
+
+Groups
+------
+
+To use the group adapter you need to register a group. For Ruby on Rails this can be done in an initializer.
+
+```ruby
+FlipFlop::Group.register(:group_name) do |actor|
+  actor.id % 2 == 0
+end
+```
+
+When checking if the feature is enabled for that group you can provide the actor (user).
+
+```ruby
+FlipFlop.feature_enabled? :feature_for_group_a, user
```

Comment:
For documentation clarity would it be better to make `: feature_for_group_a` and `: group_name` above the same value?